doc/nrf/releases_and_maturity/migration/migration_sysbuild.rst

```diff
@@ -451,9 +451,9 @@ The expected output files are the following:
 +-----------------------------------------------------+-------------------------------------------------------------------------------------------------------+
 | ``ipc_radio/zephyr/zephyr.bin``                     | Network core IPC radio binary file                                                                    |
 +-----------------------------------------------------+-------------------------------------------------------------------------------------------------------+
-| ``b0n/zephyr/zephyr.bin``                           | Network core secure bootloader hex file                                                               |
+| ``b0n/zephyr/zephyr.bin``                           | Network core secure bootloader binary file                                                            |
 +-----------------------------------------------------+-------------------------------------------------------------------------------------------------------+
-| ``b0n/zephyr/zephyr.hex``                           | Network core secure bootloader binary file                                                            |
+| ``b0n/zephyr/zephyr.hex``                           | Network core secure bootloader hex file                                                               |
```

Comment:
Hi @FrancescoSer, this seems like a typo. Can we integrate it to your PR: https://github.com/nrfconnect/sdk-nrf/pull/15760?